### ISSUE TITLE
No Menus when an editor is open on 10.15.3

### ISSUE THREAD

**Issue by jwadamson** [macvim-dev/macvim#1027]
**Describe the bug**
When no editors are open, the menu bar reads: "MacVim File Edit Window Help"
When any editors are open, the menu bar reads: "MacVim"

Losing the native save/new/copy/paste interface is very disorientating.

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Launch MacVim
2. Open/Close Editor windows

**Expected behavior**
Standard menus should be visible.

**Screenshots**
<img width="1043" alt="Screen Shot 2020-03-23 at 8 49 09 AM" src="https://user-images.githubusercontent.com/214827/77318619-bc416600-6ce3-11ea-9e44-1bb4b99b1a92.png">
<img width="882" alt="Screen Shot 2020-03-23 at 8 49 48 AM" src="https://user-images.githubusercontent.com/214827/77318595-b2b7fe00-6ce3-11ea-90d6-8713fd255d01.png">

**Environment (please complete the following information):**
 - Vim version Custom Version . (162)
 - OS: macOS 10.15.3
 - Terminal: GUI

**Comment by eirnym** [comment]
@jwadamson Thank you for an issue, 

Could you provide a bit more information about your environment?
* From which source do you use MacVim from? Github, Homebrew, MacPorts, or you've build yourself?
* Could you also provide MacVim-specific settings by running from Terminal following command: `defaults read org.vim.MacVim | grep 'MM\|SU'`

Could you also try to run your MacVim without processing `vimrc`? (`mvim --clean`) to make sure your `vimrc` or any plugins you aren't involved

**Comment by jwadamson** [comment]
Ok, closing this. 

Apparently when I moved from a manual install to brew (and simultaneously moved from having it in my home folder to Applications), I had some residual configuration path and environment variables like bashrc. For example: `VIMRUNTIME=~/Applications/MacVim.app/Contents/Resources/vim/runtime`

**Comment by jwadamson** [comment]
Thank you for your attention, and sorry if I wasted any of your time. It didn't really occur to me how other vim specific configuration could bleed into the macvim app runtime and affect the GUI like that.

**Comment by eirnym** [comment]
currently MacVim's menus are populated from scripts, so yes, moving runtime to another folder could change everything 

You're welcome

**Comment by ychin** [comment]
I’m glad you figured out the issue but I’m curious why you would need to manually set `VIMRUNTIME` in your bashrc. Most Vim installations should have it configured for you and it should be rare that you need to set it yourself.

**Comment by jwadamson** [comment]
Would have been quite a while ago, almost certain it was part of trying to shadow the default global vim command line environment with an updated userland MacVim one. I used to keep MacVim in my ~/Applications running as a standard account. 

I very well could have just been wrong to do that, but as long as all the PATH references lined up, it probably worked just fine. Guessing I had adopted a vimrc line or extension that didn't work in an older vim environment.
